weasyprint/layout/tables.py

```diff
@@ -38,6 +37,10 @@ def table_layout(context, table, max_position_y, skip_stack, containing_block,
         position_x += width
     rows_width = position_x - rows_x
 
+    # This should fix the rtl direction for table
+    if table.style['direction'] == 'rtl':
+        column_positions.reverse()
+
```

Comment:
This is actually wrong, reverting is not enough :/.

Let’s take a simple example: with a table with two columns and no borders, the first column’s width is 10px and the second one’s is 20px. If the table’s x position is 0, the ltr positions are (0, 10), and the rtl positions should be (20, 0).

```
LTR
0          10
+----------+--------------------+
| 1        | 2                  |
+----------+--------------------+

RTL
0                    20
+--------------------+----------+
| 2                  | 1        |
+--------------------+----------+
```